web/src/components/netflow-traffic-parent.tsx

```diff
@@ -47,15 +48,14 @@ class NetflowTrafficParent extends React.Component<Props, State> {
         </div>
       );
     }
-    if (this.props.children) {
-      return this.props.children;
-    }
-    // else render default NetworkTraffic
-
     return (
       <DynamicLoader>
         <AlertFetcher>
-          <NetflowTraffic forcedFilters={null} />
+          <NetflowTraffic
+            isTab={this.props.isTab}
+            forcedFilters={this.props.isTab ? this.props.forcedFilters : null}
+            parentConfig={this.props.parentConfig}
+          />
         </AlertFetcher>
       </DynamicLoader>
     );
```

Comment:
The magic happens here since I'm forcing `NetflowTrafficParent` to always load `DynamicLoader` & `AlertFetcher`.

Previously these were missing since NetflowTrafficTab was calling
```
    <NetflowTrafficParent>
      <NetflowTraffic forcedFilters={forcedFilters} isTab parentConfig={config} />
    </NetflowTrafficParent>
```